### PR TITLE
Update SupportedMods.txt

### PR DESCRIPTION
**Pull request by MikeLydeamore**
Created documentation for Applied Energistics.

### CODE REVIEW

**Comment by tterrag1098** [comment]
Does this look ok in the book?

**Comment by MikeLydeamore** [comment]
Well, there's no pages in the TPPI Guide (At least my version). It's a bit cut off at the bottom but I couldn't make it work. Otherwise it formats fine. Is it too long? Wasn't sure.

File: config/TPPI/SupportedMods.txt [hunk 1]
```diff
@@ -93,7 +93,142 @@ becomes 1 Sunnarium (needed for the ultimate hybrid solar panel) with the input
 any amount of energy (i.e. It won't explode).
 
 >AppliedEnergistics, Applied Energistics<
-<http://ae-mod.info/>
+**<http://ae-mod.info/>
+Applied Energistics is an advanced way to store items as energy in what is known as an ME system. 
+It allows for incredibly compact storage as well as item movement throughut your base.
```
Comment (tterrag1098) [inline config/TPPI/SupportedMods.txt]: Typo ;)

**Comment by tterrag1098** [comment]
There is still a book format for every mod, that you get by using /tppi mods [modname]. I have checked and while everything does fit on the page, some are not very well formatted, if you don't mind you can change to make it look better.

The cutting off at the bottom is a bug, and one I've been meaning to take a look at for a while now. 

Also, you have a typo on line 98, as commented.

Thanks for the contribution!

I'll pull this in once you tell me you're finished with it.

**Comment by MikeLydeamore** [comment]
OK, I've fixed up some formatting and typos. Let me know of anything else you'd like changed, otherwise I'm good with it.

**Pull request merged**